src/assets/noTasks.svg

```diff
@@ -0,0 +1 @@
le="evenodd" clip-rule="evenodd" d="M122.107 3.147c-1.585-2.693-9.587-4.278-12.676 0-3.629 5.025.684 9.348.684 9.348v-3.86h8.695l3.773 3.248v4.574h1.389a3.446 3.446 0 013.446-3.447c.297 0 .579.05.854.12.32.082.619.208.896.373.577-2.49.932-5.978-.81-8.138-2.136-2.648-6.251-2.218-6.251-2.218z" fill="#242424"/><path fill-rule="evenodd" clip-rule="evenodd" d="M129.168 13.503a3.418 3.418 0 00-.896-.373 3.395 3.395 0 00-.854-.12 3.446 3.446 0 00-3.446 3.447h-1.389v-4.573l-3.773-3.25h-8.695V22.69a3.449 3.449 0 004.225 3.36l4.084-1.07v8.783l4.924 4.88 4.924-4.88V19.784a3.438 3.438 0 002.593-3.327 3.434 3.434 0 00-1.697-2.954z" fill="#F5C5B6"/><path fill-rule="evenodd" clip-rule="evenodd" d="M119.2 18.604c.015.397-.068.794-.253 1.16a2.374 2.374 0 01-4.075.28l4.328-1.44zm-.435 3.219a3.652 3.652 0 001.348-1.477 3.644 3.644 0 00.206-2.802l-.206-.618-6.977 2.321.205.62a3.672 3.672 0 005.424 1.956z" fill="#242424"/><defs><linearGradient id="paint5_linear" x1="237.228" y1="101.077" x2="-.58" y2="269.656" gradientUnits="userSpaceOnUse"><stop stop-color="#FFE6CF"/><stop offset="1" stop-color="#F88"/></linearGradient><linearGradient id="paint0_linear" x1="240.214" y1="250.598" x2="230.449" y2="302.378" gradientUnits="userSpaceOnUse"><stop stop-color="#FFB36D"/><stop offset="1" stop-color="#F88"/></linearGradient><linearGradient id="paint2_linear" x1="172.092" y1="234.33" x2="152.324" y2="279.253" gradientUnits="userSpaceOnUse"><stop stop-color="#FFB36D"/><stop offset="1" stop-color="#F88"/></linearGradient><linearGradient id="paint3_linear" x1="132.599" y1="55.59" x2="61.722" y2="98.087" gradientUnits="userSpaceOnUse"><stop stop-color="#FFB36D"/><stop offset="1" stop-color="#F88"/></linearGradient><linearGradient id="paint4_linear" x1="90.815" y1="101.077" x2="-146.994" y2="269.657" gradientUnits="userSpaceOnUse"><stop stop-color="#FFE6CF"/><stop offset="1" stop-color="#F88"/></linearGradient><linearGradient id="paint1_linear" x1="175.821" y1="258.094" x2="146.469" y2="325.704" gradientUnits="userSpaceOnUse"><stop stop-color="#FFB36D"/><stop offset="1" stop-color="#F88"/></linearGradient><linearGradient id="paint6_linear" x1="287.875" y1="190.673" x2="262.24" y2="261.875" gradientUnits="userSpaceOnUse"><stop stop-color="#FFB36D"/><stop offset="1" stop-color="#F88"/></linearGradient><linearGradient id="paint7_linear" x1="236.826" y1="201.916" x2="181.71" y2="286.553" gradientUnits="userSpaceOnUse"><stop stop-color="#FFB36D"/><stop offset="1" stop-color="#F88"/></linearGradient><linearGradient id="paint8_linear" x1="141.462" y1="190.673" x2="115.827" y2="261.875" gradientUnits="userSpaceOnUse"><stop stop-color="#FFB36D"/><stop offset="1" stop-color="#F88"/></linearGradient><linearGradient id="paint9_linear" x1="90.413" y1="201.916" x2="35.297" y2="286.553" gradientUnits="userSpaceOnUse"><stop stop-color="#FFB36D"/><stop offset="1" stop-color="#F88"/></linearGradient><clipPath id="clip0"><path fill="#fff" d="M.685 52.868h240.632V293.5H.685z"/></clipPath></defs></svg>
```

Comment:
Could we rename this file `noData.svg` ? Since this svg is used also for the Disputes https://github.com/aragon/court-dashboard/blob/development/src/assets/noData.svg